HISTORY.rst

```diff
@@ -2,6 +2,23 @@
 History
 =======
 
+v3.45.0 (2021-09-30)
+--------------------
+
+Changes
+
+-  Updated the ``deploy_marketing_cloud_package`` task for compatibility with the October 2021 release of Marketing Cloud. (#2899)
+-  The ``--max-lines`` option on the ``cci error info`` command has been removed. (#2895)
+
+Issues fixed
+
+-  Removed the unused ``--skip`` option for the ``cci flow run`` command. (#2884)
+-  Flow descriptions no longer generate a warning. (#2885)
+-  We changed how the output from some commands and tasks are displayed in the CLI. (#2887)
```

Comment:
this seems more like a change than an issue fixed, but not worth holding up the release for

Reply:
Will tweak in the release body.